### PR TITLE
Fix benchmark spec

### PR DESCRIPTION
**Pull request by will**
references commit 3259b71a88fb8e72ee05797417990b0fa7498a25

### CODE REVIEW

File: spec/std/benchmark_spec.cr [hunk 1]
```diff
@@ -21,8 +21,11 @@ describe Benchmark::IPS::Job do
...
     a.mean.should be > 10
 
-    # one of the reports should be normalized to the fastest
-    a.slower.should eq(1)
+    # one of the reports should be normalized to the fastest but do to the
+    # timer precisison sleep 0.001 may not always be faster than 0.002 so we
+    # don't care which
+    first, second = [a.slower, b.slower].sort
+    first.should eq(1)
     b.slower.should be > 1
```
Comment (jhass) [inline spec/std/benchmark_spec.cr]: So this should be `second.slower`?

Comment (will) [inline spec/std/benchmark_spec.cr]: Yes thanks. It's in my editor but I forgot a `:w` :<

Comment (will) [inline spec/std/benchmark_spec.cr]: updated

**Comment by asterite** [comment]
Did you stop drinking coffee and started to c0de? :-)

**Comment by will** [comment]
My first exactly 50 commits were cafe, then I switched to c0de for the next 50 :)

**Comment by jhass** [comment]
Thanks!

**Pull request merged**